<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=1024" />
    <meta name="apple-mobile-web-app-capable" content="yes" />
    <title>impress.js | presentation tool based on the power of CSS3 transforms and transitions in modern browsers | by Bartek Szopka @bartaz</title>
    
    <meta name="description" content="impress.js is a presentation tool based on the power of CSS3 transforms and transitions in modern browsers and inspired by the idea behind prezi.com." />
    <meta name="author" content="Bartek Szopka" />

    <link href="http://fonts.googleapis.com/css?family=Open+Sans:regular,semibold,italic,italicsemibold|PT+Sans:400,700,400italic,700italic|PT+Serif:400,700,400italic,700italic" rel="stylesheet" />

    <!--
        
        Impress.js doesn't depend on any external stylesheet. Script adds all styles it needs for
        presentation to work.
        
        This style below contains styles only for demo presentation. Browse it to see how impress.js
        classes are used to style presentation steps, or how to apply fallback styles, but I don't want
        you to use them directly in your presentation.
        
        Be creative, build your own. We don't really want all impress.js presentations to look the same,
        do we?
        
        When creating your own presentation get rid of this file. Start from scratch, it's fun!
        
    -->
    <link href="css/impress-demo.css" rel="stylesheet" />
    
    <link rel="shortcut icon" href="favicon.png" />
    <link rel="apple-touch-icon" href="apple-touch-icon.png" />
</head>

<!--
    
    Body element is used by impress.js to set some useful class names, that will allow you to detect
    the support and state of the presentation in CSS or other scripts.
    
    First very useful class name is `impress-not-supported`. This class means, that browser doesn't
    support features required by impress.js, so you should apply some fallback styles in your CSS.
    It's not necessary to add it manually on this element. If the script detects that browser is not
    good enough it will add this class, but keeping it in HTML means that users without JavaScript
    will also get fallback styles.
    
    When impress.js script detects that browser supports all required features, this class name will
    be removed.
    
    The class name on body element also depends on currently active presentation step. More details about
    it can be found later, when `hint` element is being described.
    
-->
<body class="impress-not-supported">

<!--
    For example this fallback message is only visible when there is `impress-not-supported` class on body.
-->
<div class="fallback-message">
    <p>Your browser <b>doesn't support the features required</b> by impress.js, so you are presented with a simplified version of this presentation.</p>
    <p>For the best experience please use the latest <b>Chrome</b>, <b>Safari</b> or <b>Firefox</b> browser.</p>
</div>

<!--
    
    Now that's the core element used by impress.js.
    
    That's the wrapper for your presentation steps. In this element all the impress.js magic happens.
    It doesn't have to be a `<div>`. Only `id` is important here as that's how the script find it.
    
    You probably won't need it now, but there are some configuration options that can be set on this element.
    
    To change the duration of the transition between slides use `data-transition-duration="2000"` giving it
    a number of ms. It defaults to 1000 (1s).
    
    You can also control the perspective with `data-perspective="500"` giving it a number of pixels.
    It defaults to 1000. You can set it to 0 if you don't want any 3D effects.
    If you are willing to change this value make sure you understand how CSS perspective works:
    https://developer.mozilla.org/en/CSS/perspective
    
    But as I said, you won't need it for now, so don't worry - there are some simple but interesing things
    right around the corner of this tag ;)
    
-->
<div id="impress">

    <!--
        
        Here is where interesting thing start to happen.
        
        Each step of the presentation should be an element inside the `#impress` with a class name
        of `step`. These step elements are positioned, rotated and scaled by impress.js, and
        the 'camera' shows them on each step of the presentation.
        
        Positioning information is passed through data attributes.
        
        In the example below we only specify x and y position of the step element with `data-x="-1000"`
        and `data-y="-1500` attributes. This means that **the center** of the element (yes, the center)
        will be positioned in point x = -1000px and y = -1500px of the presentation 'canvas'.
        
        It will not be rotated or scaled.
        
    -->
    <div id="intro" class="step slide" data-x="-0000" data-y="-1500">
        <p style="text-align:right;color:blue;font-size:small" id="pZ"><a href="#theEnd">+</a></p>
        <i style="font-size:medium">Informatics Division of the Department of Pathology at UAB</i>
        <hr style="border-color:navy">
        <p style="font-size:xx-large;color:navy">Informatics for Systems Biomedicine</p>
        <p style="font-size:x-large;color:navy">Data sciences for context dependent computing</p>
        <p style="color:navy;font-size:large"><a href="http://jonasalmeida.info" target=_blank>Jonas S Almeida (<span style="color:blue;font-size:large"> jonasalmeida.info</span></a>)</i>
        </p>
        <hr style="border-color:navy">
        <p>&nbsp;</p>
        <div id="summary"> 
        <!--  style="text-indent:-30px"
        &nbsp;
        <h2>Summary:</h2>
        &nbsp;
        <p>1. The one computer world</p>
        <p>2. Scale and reproducibility</p>
        <p>3. Sustainable software ecosystems and interoperability</p>
        <p>4. Outline of the division of Informatics</p>
        <p>5. Data Sciences at UAB.</p> -->
        <p>Data Sciences that follow the biomedical data !</p>
        <p>&nbsp;</p>
        <li>Pathology</li>
        <li>Cancer Center</li>
        <li>Center for Clinical Translation Sciences</li>
        <li>UAB-wide Data Science Program</li>
        <li>Biomed BigData is the opportunity and the challange of user-facing bioinformatics.</li>
        </div>
        
        

    </div>

    <div id="dataScience" class="step slide" data-x="-0000" data-y="-1500" data-z="1500">
        <p style="text-align:right;color:blue;font-size:small"><a href="#theEnd">+</a></p>
        <hr>
        <p>
            <i>" ...the broad area encompassed by data science (i.e., bioinformatics, computational biology, biomedical informatics, biostatistics, information science, quantitative biology, etc.)"*</i>
        </p>
        <hr>
        <p>&nbsp;</p>
        &nbsp;
        <p>
            * NIH Data Science Initiative
        </p>
        <p>&nbsp;</p>

        <p>
            <li><a href="http://biomedicalcomputationreview.org/content/nih-announcement-big-data-gets-big-support" target=_blank>NIH BigData to Knowledge (BD2K)</a>.</li>
        </p>
        &nbsp;
        <p>
            <li><a href="http://www.whitehouse.gov/blog/2013/04/23/big-data-big-deal-biomedical-research" target=_blank> Office of Science and Technology</a></li>
        </p>
        &nbsp;

        <p>
            <li>NSF <a href="http://www.nsf.gov/funding/pgm_summ.jsp?pims_id=504767" target=_blank>BIGDATA</a> and <a href="http://www.nsf.gov/news/news_summ.jsp?cntn_id=125610" target=_blank>others</a>.</li>
        </p>
    </div>


    <div id="informaticsDiv" class="step slide" data-x="-1000" data-y="-1500" data-z="-1500">
        <p style="text-align:right;color:blue;font-size:small"><a href="#theEnd">+</a></p>
        <!--<p style="font-size:x-large;color:navy">Informatics Division</p>-->
        <p style="font-size:medium;color:navy">Created January 2011, planned for ~4-5 faculty, full strength July 2013</p>
        <hr style="border-color:navy">
        <a href="http://www.uab.edu/medicine/pathology/divisions-a-sections/informatics" target=_blank>Division of Informatics at the Dept of Pathology<br>
        <img src="divInformatics.png"></a>
        <!--<iframe src="http://uab.mathbiol.org/informatics" width=100% height=500></iframe>-->
    </div>

    <div id="JonasAlmeida" class="step slide" data-x="-1000" data-y="-750" data-z="-1500">
        <p style="text-align:right;font-size:large"><i>Informatics Division of the Department of Pathology at UAB </i><a href="#theEnd" style="color:blue"> + </a></p>
        <table style="padding:20px">
            <tr>
                <td style="padding:20px">
                    <img src="jonas2013crop.jpeg" width=150>
                </td>
                <td style="color:navy;font-size:x-large;padding:20px;vertical-align:top">
                    <li style="color:maroon;font-weight:bold"> Jonas Almeida, PhD, Professor, Director</li>
                    <p style="font-size:small">&nbsp;</p>
                    <p><a href="http://ibl.github.io/" target=_blank>Integrative Bioinformatics Lab</a></p>
                    <p style="color:navy;font-size:medium;line-height:1.5">My research is focused on the computational infrastructure and Data Science underlying integrative bioinformatics. Specifically, it seeks to articulate the computational statistics aspects of data analysis with the data representation and management of its acquisition for systems level modeling. Ongoing work is focused on applications Big Data such as The Cancer Genome Atlas (TCGA), and clinical environments, such as Pathology image analysis workflows.</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Kim YW, Koul D, Kim SH, Lucio-Eterovic AK, Freire PR, Yao J, Wang J, Almeida JS, Aldape K, Yung WK. (2013) "Identification of prognostic gene signatures of glioblastoma: a study based on TCGA data analysis." Neuro-Oncology (in press) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23502430" target=_blank>PMID:23502430</a>].</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Robbins DE, A Gruneberg, HF Deus, MM Tanik, JS Almeida (2013) A Self-Updating Roadmap of The Cancer Genome Atlas. Bioinformatics [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23595662" target=_blank>PMID:23595662</a>].</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Almeida JS, A Grüneberg, W Maass, S Vinga (2012) Fractal MapReduce decomposition of sequence alignment. Algorithms for Molecular Biology 7:12 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22551205" target=_blank>PMID:22551205</a>].</p>
                </td>
            </tr>
        </table>
    </div>

    <div id="ibl" class="step slide" data-x="-1000" data-y="0" data-z="-1500">
        <iframe src="http://ibl.github.io" width=100% height=600></iframe>
    </div>

    <div id="MalayBasu" class="step slide" data-x="-0150" data-y="-1500" data-z="-1500" data-rotate="-90">
        <p style="text-align:right;font-size:medium"><i>Informatics Division of the Department of Pathology at UAB </i><a href="#theEnd" style="color:blue"> + </a></p>
        <table style="padding:20px">
            <tr>
                <td style="padding:20px">
                    <img src="MalayBasu.jpg" width=150>
                </td>
                <td style="color:navy;font-size:x-large;padding:20px;vertical-align:top">
                    <li style="color:maroon;font-weight:bold"> Malay Basu, PhD, Assistant Professor</li>
                    <p style="font-size:small">&nbsp;</p>
                    <p><a href="http://ibl.github.io/" target=_blank>Computational Genomics Lab</a></p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Genomes as the “book of life”. My research focus is to decipher genomes literally as a book: I use computational linguistics to identify linguistic “universals” in genomes to investigate how genome organization affects evolution.  Using data from sequencing initiatives, I study how the gramar of cancer genomes evolve and look for evolutionary patterns. I also participate in several collaborative research efforts, such as investigating roles of mitochondrial morphology in malignant transformation.</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Haft D.H., Selengut J.D., Richter R.A., Harkins D., Basu M.K., and Beck E. TIGRFAMS and Genome Properties in 2013 (2013) Nucleic Acids Res, 41(Database issue):D387-95. doi: 10.1093/nar/gks1234. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23502430" target=_blank>PMID:22551205</a>].</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Basu M. K., Selengut J. D., and Haft D. H. (2011) ProPhylo: partial phylogenetic profiling to guide protein family construction and assignment of biological process. BMC Bioinform, 12(1):434. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23502430" target=_blank>PMID:23595662</a>].</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Haft D. H. and Basu M. K. (2011) Biological systems discovery in silico: radical S- adenosylmethionine protein families and their target peptides for posttranslational modification. J. Bacteriol., 193(11):2745–2755. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23502430" target=_blank>PMID:23502430</a>].</p>

                </td>
            </tr>
        </table>
    </div>

    <div id="EmidioCapriotti" class="step slide" data-x="-1000" data-y="-2250" data-z="-1500" data-rotate="-180">
        <p style="text-align:right;font-size:large"><i>Informatics Division of the Department of Pathology at UAB </i><a href="#theEnd" style="color:blue"> + </a></p>
        <table style="padding:20px">
            <tr>
                <td style="padding:20px">
                    <img src="EmidioCapriotti.png" width=150>
                </td>
                <td style="color:navy;font-size:x-large;padding:20px;vertical-align:top">
                    <li style="color:maroon;font-weight:bold"> Emidio Capriotti, PhD, Assistant Professor</li>
                    <p style="font-size:small">&nbsp;</p>
                    <p><a href="http://ibl.github.io/" target=_blank>Computational Molecular Biology</a></p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Study of the effect of SNPs resulting in single amino acid polymorphisms. I develop machine learning-based tools to predict the effect of single point protein mutations on the protein stability (I-Mutant) to estimate the free energy change due to a given single amino acid polymorphisms (SAPs). The increasing amount of variation data made available by the 1000 Genomes and The Cancer Genome Atalas (TCGA) consortiums, allow for new disease-specific algorithms and tools for personalized medical genomics solutions.</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Capriotti E, Altman RB, Bromberg Y. (2013) Collective judgment predicts disease-associated single nucleotide variants. BMC Genomics. 14 (Suppl 3): S2. (in press).</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Kemena C, Bussotti G, Capriotti E, Marti-Renom MA, Cedric Notredame C. (2013). Using tertiary structure for the computation of highly accurate multiple RNA alignments with the SARA-Coffee package. Bioinformatics. 29:1112-1119. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23449094" target=_blank>PMID:23449094</a>].</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Capriotti E, Altman RB. (2011). A new disease-specific machine learning approach for the prediction of cancer-causing missense variants. Genomics. 98; 310-317. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21763417" target=_blank>PMID:21763417</a>].</p>

                </td>
            </tr>
        </table>
    </div>

    <div id="emidioBiofold" class="step slide" data-x="-1000" data-y="-3000" data-z="-1500" data-rotate="-180">
        <iframe src="http://biofold.org/emidio" width=100% height=600></iframe>
    </div>
    

    <div id="RobSeung" class="step slide" data-x="-1850" data-y="-1500" data-z="-1500" data-rotate="-270">
        <p style="text-align:right;font-size:large"><i>Informatics Division of the Department of Pathology at UAB </i><a href="#theEnd" style="color:blue"> + </a></p>
        <table style="padding:20px">
            <tr>
                <td style="padding:20px">
                    <img src="RobHackney.jpeg" width=150>
                </td>
                <td style="color:navy;font-size:x-large;padding:20px;vertical-align:top">
                    <li style="color:maroon;font-weight:bold"> Rob Hackney, MD, Assistant Professor</li>
                    <p style="font-size:small">&nbsp;</p>
                    <p><a href="http://ibl.github.io/" target=_blank>NeuroPathology</a>
                    <span style="color:navy;font-size:medium;line-height:1.5"> and Pathology Informatics</span></p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Almeida JS, E Iriabho, VL Gorrepati, S Wilkinson, DE Robbins, A Grüneberg, JR Hackney (2012) ImageJS: personalized, participated, pervasive and reproducible image bioinformatics in the web browser. J Pathology Informatics 3:25 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22934238" target=_blank>PMID:22934238</a>].</p>
                    
                </td>
            </tr>
            <tr>
                <td style="padding:20px">
                    <img src="SeungPark.png" width=150>
                </td>
                <td style="color:navy;font-size:x-large;padding:20px;vertical-align:top">
                    <li style="color:maroon;font-weight:bold"> Seung Park, MD, Assistant Professor</li>
                    <p style="font-size:small">&nbsp;</p>
                    <p><a href="http://ibl.github.io/" target=_blank>Pathology Informatics</a>
                    <span style="color:navy;font-size:medium;line-height:1.5"> and Anatomic Pathology</span></p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Park S, Pantanowitz L, Parwani A, Wells A, Oltvai Z.  “Workflow Organization in Pathology”. Clinics in Laboratory Medicine 32(2012) 601-622.[<a href="http://www.ncbi.nlm.nih.gov/pubmed/23078662" target=_blank>PMID:23078662</a>].</p>
                    <p style="font-size:small">...</p>
                    <p style="color:navy;font-size:medium;line-height:1.5">Park S, Pantanowitz L, Sharma G, Parwani A.  “Anatomic Pathology Laboratory Information Systems: A Review”.  Advances in Anatomic Pathology 2012 March; 19(2): 81-96.[<a href="http://www.ncbi.nlm.nih.gov/pubmed/22313836" target=_blank>PMID:22313836</a>].</p>
                </td>
            </tr>
        </table>
    </div>

    <div id="PathologyPersonalizedMedicine" class="step slide" data-x="-1000" data-y="-1500" data-rotate="-270">
        <i>May 2011</i><br><img src="PathologyPersonalizedMedicine.png" width="100%">
    </div>

    <div id="pathologySubspeciality" class="step slide" data-x="-1800" data-y="-1500" data-rotate="-270">
        <i>March 2013</i><hr>
        <img src="pathologySubspeciality.png" width="100%">
    </div>

    <div id="tcga" class="step slide" data-x="-2600" data-y="-1500" data-rotate="-270">
        <a href="https://tcga-data.nci.nih.gov/tcga/" target=_blank><img src="https://tcga-data.nci.nih.gov/tcga/images/general/tcga-dp_logo.png"></a><br>
        <iframe src="https://tcga-data.nci.nih.gov/tcga/" width=100% height=550></iframe>
    </div>

    <div id="tcgaRoadMap" class="step slide" data-x="-2600" data-y="-1500" data-z="-1500" data-rotate="-270">
        <i style="color:navy"><span style="font-size:large"><a href="http://tcga.github.com/Roadmap/" target=_blank>A Self-Updating Roadmap of The Cancer Genome Atlas</a></span><br><span style="font-size:medium">Robbins DE, A Gruneberg, HF Deus, MM Tanik, JS Almeida (2013) Bioinformatics 4(17) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23595662" target=_blank>PMID 23595662</a>]</span></i><br>
        <iframe src="http://tcga.github.com/Roadmap" width=100% height=550></iframe>
    </div>

    <div id="imagejs" class="step slide" data-x="-2600" data-y="-1500" data-z="-3000" data-rotate="-270"><span style="font-size:small;color:navy;line-height:0" >
        Almeida JS, E Iriabho, VL Gorrepati, S Wilkinson, DE Robbins, A Grüneberg, JR Hackney (2012) J Pathology Informatics 3:25 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22934238" target=_blank>PMID:22934238</a>]<br>ImageJS: personalized, participated, pervasive and reproducible image bioinformatics in the web browser.</span><hr>
        <iframe src="http://imagejs.org" width="100%" height="450"></iframe>
        <span style="font-size:medium">Example of active link: <a href="https://bitly.com/withData" target=_blank><img src="bitlyWithdata.png" width=50></a></span>
    </div>

    <div id="systemsBiomedicine" class="step slide" data-x="-2600" data-y="-1500" data-z="-4500" data-rotate="-270">
        <p style="text-align:right;font-size:medium">div Informatics <a style="color:blue" href="#theEnd"> + </a></p>
        <img src="systemsBiomedicine.png" width="100%">
    </div>

    <div id="dataDeluge" class="step slide" data-x="-3300" data-y="-1500" data-z="-4500" data-rotate="-270">
        <img src="beyondDataDeluge.png" width="90%">
    </div>

    <div id="Kinomics" class="step slide" data-x="-4000" data-y="-1500" data-z="-4500" data-rotate="-270" style="color:navy">
        <a href="http://adussaq.github.com/kinomicDataQC/" target=_blank>Kinomics webApp<hr>
        <img src="kinomicsUAB.png" width="90%"></a>
    </div>

    <div id="Conclusions" class="step slide" data-x="-4700" data-y="-1500" data-z="-4500" data-rotate="-270" style="color:navy">
        <p style="text-align:right;font-size:medium">div Informatics <a style="color:blue" href="#theEnd"> + </a></p>
        <p><b>Conclusion</b></p>
        <hr>
        <li>Division at full strength July 2013 - assembly phase ends, development stage begins.</li>
        <p>&nbsp;</p>
        <li>Translational, training, and bioinformatics focus on a <b style="color:maroon;background-color: #ffff00"> Digital Tumor Board </b>.</li>
        <p>&nbsp;</p>
        <li> $: 1) New external funding opportunities emmerging, such as NIH BD2K program, but need for 2) clinical integration (service) and 3) entrepreneurial venues (start-ups).</li>
        <p>&nbsp;</p>
        <li> Challenge: translational development feeding scholarly work.</li>
        <p>&nbsp;</p>
        <li> "Racing towards reimbursement".</li>
        <p>&nbsp;</p>

    </div>

    <div id="theEnd" class="step" data-x="0000" data-y="-1000" data-z="460" data-rotate-x="-40" data-rotate-y="-5" data-scale="6">
    </div>



<script>
if ("ontouchstart" in document.documentElement) { 
    document.querySelector(".hint").innerHTML = "<p>Tap on the left or right to navigate</p>";
}
</script>

<!--
    
    Last, but not least.
    
    To make all described above really work, you need to include impress.js in the page.
    I strongly encourage to minify it first.
    
    In here I just include full source of the script to make it more readable.
    
    You also need to call a `impress().init()` function to initialize impress.js presentation.
    And you should do it in the end of your document. Not only because it's a good practice, but also
    because it should be done when the whole document is ready.
    Of course you can wrap it in any kind of "DOM ready" event, but I was too lazy to do so ;)
    
-->
<script src="js/impress.js"></script>
<script>impress().init();</script>

<!--
    
    The `impress()` function also gives you access to the API that controls the presentation.
    
    Just store the result of the call:
    
        var api = impress();
    
    and you will get three functions you can call:
    
        `api.init()` - initializes the presentation,
        `api.next()` - moves to next step of the presentation,
        `api.prev()` - moves to previous step of the presentation,
        `api.goto( idx | id | element, [duration] )` - moves the presentation to the step given by its index number
                id or the DOM element; second parameter can be used to define duration of the transition in ms,
                but it's optional - if not provided default transition duration for the presentation will be used.
    
    You can also simply call `impress()` again to get the API, so `impress().next()` is also allowed.
    Don't worry, it wont initialize the presentation again.
    
    For some example uses of this API check the last part of the source of impress.js where the API
    is used in event handlers.
    
-->

</body>
</html>


